src/components/connect.js

```diff
@@ -158,6 +159,11 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
           window.clearTimeout(this.state.refreshTimeouts[prop])
         }
 
+        if (mapping.value) {
+          this.setAtomicState(prop, startedAt, mapping, PromiseState.resolve(mapping.value), null)
+          return
```

Comment:
just realized this `return` here will block chaining off of an identity request. Not really sure what the use case would be for that, but for consistency, identity requests should still support chaining. need to fix.